index.rst

```diff
@@ -48,7 +48,353 @@
+
+* They will be a subset both in rows and columns of what the full ``Source`` table will contain in a Data Release.
+* They will be provided in a Parquet form following the application of some level of "SDM Standardization"-like processing.
+* They will be accessed via the Notebook Aspect as a Butler dataset, with the same DataID of a CCD visit image.
+
+Other considerations will be discussed below.
+
+
+Spectroscopic Data
+------------------
+
+At some point the AuxTel/LATISS spectroscopic data may be run through an AP-like pipeline,
+up through the ISR stage,
+and these results would be readily made available through the "PP Preview" system as well.
+Whether this will actually be done is TBD, and whether the actual spectroscopic reduction outputs, or
+any resulting atmospheric model updates, would be available via "PP Preview" remains to be determined as well.
+
+
+Data Access
+===========
+
+This section covers the ways in which data from the processing above would be made available to the
+(internal) RSP users of "PP Preview".
+
+Overview
+--------
+
+The AuxTel and subsequent processed datasets described above are meant to be made available in the RSP
+in a way which progressively
+approaches exactly how LSSTCam Prompt Products will be made available to science users during operations.
+This will follow the Science Platform Vision's model where data are available as uniformly as
+possible across the API, Notebook, and Portal Aspects.
+
+Butler Access
+-------------
+
+What may evolve over the course of "PP Preview" is that early on access may be only through
+the ``/repo/embargo`` repository, while by the end of "PP Preview" we'll have to have the
+full "release from embargo to (staff) users at 80 hours" working, and therefore have the Butler
+that represents the latter dataset, ``/repo/prompt``, available in the RSP.
+
+
+RSP Data Services (API Aspect)
+------------------------------
+
+
+Image Data
+^^^^^^^^^^
+
+As in DP0.2, image data access through the API Aspect depends on several elements:
+
+* ObsCore image metadata queries via TAP
+* A DataLink "links service" endpoint to connect image dataset IDs from ObsCore queries
+* Image-retrieval signed URLs for the actual image access, returned from the "links service"
+* An image-cutout service pointed to by metadata from the "links service"
+
+
+CAOM2 metadata
+""""""""""""""
+
+It's not a primary goal of PP Preview to exercise the creation and provision of CAOM2 metadata for
+acquired data, but it would be a useful stretch goal.
+
+Architecture-team guidance for how this metadata will be created and stored in a database will
+be required in order for progress to be made in this area.
+
+Catalog Data
+^^^^^^^^^^^^
+
+The final version of PP Preview will have to demonstrate the ability to migrate data from
+the APDB to the PPDB.
+The PPDB is expected to be a Postgres database.
+It is not completely clear whether it's the same database as will be used for the image metadata.
+
```

Comment:
I believe there should also be a subsection on the cutout service here, with mention that it is currently possible to retrieve an individual cutout for a source, but not cutouts for the full history of the associated object. A brief description of what is missing to enable cutout history would be good to include.